src/main/java/me/ryanhamshire/griefprevention/task/CleanupUnusedClaimsTask.java

```diff
@@ -83,50 +82,35 @@ public void run() {
                 Instant claimLastActive = claim.getInternalClaimData().getDateLastActive();
 
                 try (final CauseStackManager.StackFrame frame = Sponge.getCauseStackManager().pushCauseFrame()) {
-                    Sponge.getCauseStackManager().addContext(GPContextKeys.CHEST_CLAIM_EXPIRED, GriefPreventionPlugin.instance.pluginContainer);
-                    // if this claim is a chest claim and those are set to expire
-                    Double
-                        claimExpirationChest =
-                        GPOptionHandler.getClaimOptionDouble(playerData.getPlayerSubject(), claim, GPOptions.CLAIM_EXPIRATION_CHEST, playerData);
-                    if (claim.getArea() <= areaOfDefaultClaim && claimExpirationChest > 0) {
-                        if (claimLastActive.plus(Duration.ofDays(claimExpirationChest.intValue()))
-                            .isBefore(Instant.now())) {
-                            Sponge.getCauseStackManager().addContext(GPContextKeys.CHEST_CLAIM_EXPIRED, GriefPreventionPlugin.instance.pluginContainer);
 
-                            claim.removeSurfaceFluids(null);
-                            claimManager.deleteClaim(claim);
-
-                            // if configured to do so, restore the land to natural
-                            if (GriefPreventionPlugin.instance.claimModeIsActive(worldProperties, ClaimsMode.Creative) || activeConfig
-                                .getConfig().claim.claimAutoNatureRestore) {
-                                GriefPreventionPlugin.instance.restoreClaim(claim, 0);
-                            }
-
-                            GriefPreventionPlugin.addLogEntry(" " + claim.getOwnerName() + "'s new player claim " + "'" + claim.id + "' expired.",
-                                CustomLogEntryTypes.AdminActivity);
-                        }
+                    int claimExpiration = 0;
+                    EventContextKey context = GPContextKeys.PLAYER_CLAIM_EXPIRED;
```

Comment:
Should new Context Keys need be created for Basic/Town/Subdivision claim expiration?